python/ray/train/v2/_internal/execution/worker_group/worker.py

```diff
@@ -141,6 +144,8 @@ def poll_status(self) -> WorkerStatus:
             training_result = None
 
         error = execution_context.training_thread_runner.get_error()
+        if error:
+            logger.error(f"Error in training function:\n{error}")
```

Comment:
This prints the error stacktrace due to the `UserExceptionWithTraceback` wrapper that we add right?

Reply:
Oh yes you are right! Updated to the location you linked above and just printed out the stacktrace directly.